### PR TITLE
Move VERSION inside ./src

### PR DESCRIPTION
**Pull request by bcardiff**
Move VERSION inside ./src due to https://github.com/crystal-lang/crystal/pull/7784#issuecomment-494387548 

Fixes https://github.com/crystal-lang/distribution-scripts/issues/35
Fixes https://github.com/crystal-lang/crystal/issues/7602
Closes https://github.com/crystal-lang/crystal/pull/7784

### CODE REVIEW

File: bin/ci [hunk 1]
```diff
@@ -98,7 +98,7 @@ prepare_build() {
 
 verify_version() {
   # If building a tag, check it matches with file
-  FILE_VERSION=`cat VERSION`
+  FILE_VERSION=`cat ./src/VERSION`
```
Comment (j8r) [inline bin/ci]:
```suggestion
  FILE_VERSION="$(cat ./src/VERSION)"
```

Comment (asterite) [reply bin/ci]: Why?

Comment (asterite) [reply bin/ci]: I mean, it was using backticks before and it was working fine. Why risk changing it to something that apparently works the same but might break in some other subtle ways?

Comment (oprypin) [reply bin/ci]: Even if it is a positive change, I would keep it separate from this PR as unrelated.

Comment (bcardiff) [reply bin/ci]: Ok, commit removed.

**Comment by RX14** [comment]
:heart:

**Comment by j8r** [comment]
Any reason why the version isn't directly a constant inside the code?

**Comment by asterite** [comment]
I was thinking the same thing. In fact, everything was good before introducing this file.

However, I think having it in a separate file makes the commit history for other files a bit cleaner.

**Comment by oprypin** [comment]
But why not have it in a *separate* *source file*? Isn't it best of both worlds?

**Comment by asterite** [comment]
Yes, I think that makes sense. I think (but I don't know) the idea of the VERSION file was also to have it in the root directory so it's visible, but we even lost that. At this point I think it should be in code (except that it's a bit harder to read externally).

**Comment by j8r** [comment]
It can have a file with only `Crystal::VERSION = "0.28.0"`, then require it.
Easy to parse for other tools outside Crystal, too.

**Pull request merged**